src/components/IOUConfirmationList.js

```diff
@@ -133,9 +134,9 @@ class IOUConfirmationList extends Component {
                 ),
             }),
         };
-        if (this.props.iouType === CONST.IOU.IOU_TYPE.SEND && this.props.participants.length === 1) {
+        if (this.props.iouType === CONST.IOU.IOU_TYPE.SEND && this.props.participants.length === 1 && Permissions.canUseIOUSend(this.props.betas)) {
             // Add the Expensify Wallet option if available and make it the first option
-            if (this.props.localCurrencyCode === CONST.CURRENCY.USD || Permissions.canUsePayWithExpensify(this.props.betas)) {
+            if (this.props.localCurrencyCode === CONST.CURRENCY.USD && Permissions.canUsePayWithExpensify(this.props.betas)) {
```

Comment:
Do you think we should also check for canUseWallet here since otherwise they won't be able to see their balance?

Reply:
Since someone can send a payment with `Venmo`, `PayPal`, or `Settle Manually` I don't think we should block on `canUseWallet` here.

Reply:
Yeah, think that's a good idea